### PR TITLE
check out active cases over time

### PR DESCRIPTION
**Pull request by nmmarquez**
Code for looking at facilities which have been consistently reporting active case data for the past 10 months (~900 facilities) and plotting the total number of active cases at the earliest reporting date for that month. I think this gives us a decent picture of the change in infection rate among carceral facilities who report this data and highlight that the recent outbreaks due to omicron are much higher than what we saw with delta.

### CODE REVIEW

File: ad-hoc/review_active_cases.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    na.omit() %>%
+    # for each date floor to the nearest month
+    mutate(Month = floor_date(Date, unit = "month")) %>%
+    # group by each facility and each month
+    group_by(Facility.ID, Month) %>%
+    # only get the first monthly observation for each facility
+    filter(1:n() == 1) %>%
+    # only get data that fall within these date ranges
+    filter(Month >= ymd("2021-04-01")) %>%
+    filter(Month <= ymd("2022-01-01")) %>%
```
Comment (ssciolist) [inline ad-hoc/review_active_cases.R:23]: Can this date be a variable to make this more reusable? Maybe the variable defaults to system date?

File: ad-hoc/review_active_cases.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    # only keep facilities that have 10 months of observations
+    group_by(Facility.ID) %>%
+    mutate(nmonth = n()) %>%
+    filter(nmonth == 10)
```
Comment (ssciolist) [inline ad-hoc/review_active_cases.R:27]: If you can shift the filter above to be dynamic, I think we'd want to change this to not be hard-coded as well.

Comment (nmmarquez) [reply ad-hoc/review_active_cases.R:27]: Yup ill shift it to be the max number among all facilities.

**Comment by ssciolist** [comment]
That runs locally for me and I appreciate the changes!

**Pull request merged**